src/main/java/dk/kb/labsapi/SummariseExport.java

```diff
@@ -0,0 +1,239 @@
+    private String extractRecord(String soapXML, String uuid) {
+        try {
+            return XMLStepper.evaluateFakeXPath(soapXML, "Envelope/Body/getLegacyRecordResponse/getLegacyRecordReturn");
+        } catch (XMLStreamException e) {
+            log.warn("Unable to parse SOAP response for uuid '" + uuid + "'", e);
+            throw new RuntimeException("Unable to parse SOAP XML response");
+        }
+    }
+
+    /**
+     * Takes a Fedora XML and extracts the content of the first {@code ALTO} datastream.
+     */
+    //  <dobundle:digitalObjectBundle xmlns:dobundle="http://doms.statsbiblioteket.dk/types/digitalobjectbundle/default/0/1/#">
+    //    <foxml:digitalObject xmlns:foxml="info:fedora/fedora-system:def/foxml#" xmlns:xsi="http://www.w3.org/2001...
+    //     ...
+    //      <foxml:datastream CONTROL_GROUP="M" ID="ALTO" STATE="A" VERSIONABLE="true">
+    //        <foxml:datastreamVersion ALT_IDS="B400026957719-RT1/400026957719-01/1857-04-11-01/dannevirke1838-1857-04-11-01-0173B.alto.xml" CREATED="2016-04-09T20:27:54.248Z" ID="ALTO.0" LABEL="" MIMETYPE="text/xml" SIZE="259901">
+    //           <foxml:xmlContent>
+    //                  <alto xmlns="http://www.loc.gov/standards/alto/ns-v2#" xsi:schemaLocation="http://www.loc.gov/standards/alto/ns-v2# alto-v2.0.xsd">
+    private String extractALTOFromFedoraXML(String fedoraXML, String uuid) {
+        final String xPath = "digitalObjectBundle/digitalObject/datastream[@ID='ALTO']/datastreamVersion/xmlContent/alto";
+        //final String xPath = "digitalObjectBundle/digitalObject/datastream/datastreamVersion/xmlContent";
+        try {
+            XMLStreamReader xmlReader = XMLStepper.jumpToNextFakeXPath(fedoraXML, xPath);
+            if (xmlReader == null) {
+                String message = "Got response for record but could not locate ALTO for uuid '" + uuid + '"';
+                log.warn(message);
+                throw new InternalServiceException(message);
+            }
+            return XMLStepper.getSubXML(xmlReader, true);
+        } catch (XMLStreamException e) {
+            log.warn("Unable to extract ALTO from record for uuid '" + uuid + "'", e);
+            throw new InternalServiceException("Unable to extract ALTO from record for uuid '" + uuid + "'");
+        }
+    }
+
+    // Returns a recordID or throws an exception
+    private String uuidToRecordID(String id, String uuid) {
+        String query = String.format(
+                Locale.ROOT,
+                // doms_newspaperCollection:uuid:1620bf3b-7801-4a34-b2b9-fd8db9611b76-segment_19
+                "recordID:\"doms_newspaperCollection:uuid:%1$s\" OR \n" +
+                // https://www2.statsbiblioteket.dk/mediestream/avis/record/doms_aviser_page:uuid:a9990f12-e9f0-4b1e-becc-e0d4bf514586/query/heste)
+                "pageUUID:\"doms_aviser_page:uuid:%1$s\"",
+                uuid);
+        String filter = "recordBase:doms_aviser_page AND py:[* TO 1882]";
```

Comment:
Here you are extending py to 1882, which makes sense! Should we do this for other exports as well, currently export/aviser is restricted to 1880.

Reply:
I'll change 1882 to 1880 so that everything is synced.